src/main/java/mServer/crawler/sender/orf/tasks/OrfDayTask.java

```diff
@@ -0,0 +1,36 @@
+package mServer.crawler.sender.orf.tasks;
+
+import java.util.concurrent.ConcurrentLinkedQueue;
+import mServer.crawler.sender.MediathekReader;
+import mServer.crawler.sender.orf.CrawlerUrlDTO;
+import mServer.crawler.sender.orf.TopicUrlDTO;
+import org.jsoup.nodes.Document;
+import org.jsoup.select.Elements;
+
+public class OrfDayTask extends AbstractDocumentTask<TopicUrlDTO, CrawlerUrlDTO> {
+
+  private static final String ITEM_SELECTOR = "article.item > a";
+  private static final String ATTRIBUTE_HREF = "href";
+
+  public OrfDayTask(final MediathekReader aCrawler,
+          final ConcurrentLinkedQueue<CrawlerUrlDTO> aUrlToCrawlDTOs) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "aUrlToCrawlDTOs" object should be an interface such as "Queue" rather than the implementation "ConcurrentLinkedQueue". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)